homeassistant/components/mystrom/light.py

```diff
@@ -121,6 +132,16 @@ def is_on(self):
         """Return true if light is on."""
         return self._state
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info for the light entity."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self._mac)},
+            name=self.name,
+            manufacturer=ATTR_MANUFACTURER,
+            sw_version=self._bulb.firmware,
+        )
+
```

Comment:
This can be added as `self._attr_device_info` in `__init__` method.

Reply:
done